### ISSUE TITLE
Integrate into Laravel

### ISSUE THREAD

**Issue by subdesign** [sasanrose/phpredmin#65]
What do you think if I can integrate phpredmin into my Laravel 4/5 cms administration ? If yes, do you have some hints for doing that? As I see phpredmin is a little mvc framework, could it work as a single "library"? 
thanks

**Comment by sasanrose** [comment]
Can you explain more what are you exactly planing to do?

Thanks

**Comment by subdesign** [comment]
Yes. I want to use phpredmin in my administration area, if its possible. So I make a button in the admin menu system for it (behind an ACL system), and for instance it's uri would be /admin/phpredmin and the Laravel router would route to a controller from this address, which loads all the things needed to show and use phpredmin. If your solution isn't so modularized, or very unique, then it was just an idea by me..

**Comment by ctrlaltdylan** [comment]
Sounds like there's room for a fork and new composer package for something like this.

**Comment by mmucklo** [comment]
Would be interested in the same for symfony
